Example/Podfile

```diff
@@ -15,10 +15,7 @@ source 'https://github.com/artsy/Specs.git'
 source 'https://github.com/CocoaPods/Specs.git'
 
 inhibit_all_warnings!
-platform :ios, '9.0'
-
-# FIXME: Figure out why we are getting hit by this issue.
-install! 'cocoapods', deterministic_uuids: false
```

Comment:
This didn't seem necessary anymore – others have reported not needing it, but I wanted to highlight this change /cc @alloy